meta-balena-common/recipes-core/systemd/timeinit/fake-hwclock

```diff
@@ -0,0 +1,75 @@
+#!/bin/sh
+#
+# Trivial script to load/save current contents of the kernel clock
+# from/to a file. Helpful as a *bootstrap* clock on machines where
+# there isn't a useful RTC driver (e.g. on development boards). Using
+# NTP is still recommended on these machines to get to real time sync
+# once more of the system is up and running.
+#
+# Copyright 2012-2016 Steve McIntyre <93sam@debian.org>
+# Copyright 2020 Balena Ltd.
+#
+# License: GPLv2, see COPYING
+
+FILE=/etc/fake-hwclock/fake-hwclock.data
+TIMESTAMP=/etc/timestamp
+
+# Build time is used as a sanity check when saving
+if [ ! -f $TIMESTAMP ]; then
+   echo "[ERROR] $TIMESTAMP not found."
```

Comment:
You can use `os-helpers` to deal with the `[TAG]` prefixes. Just make the recipe depend on `os-helpers-logging` and then you can do
```bash
. /usr/libexec/os-helpers-logging

info "Message 1"
error "Message 2"
etc.
```
The source is in `meta-balena-common/recipes-support/os-helpers/os-helpers/os-helpers-logging`

Reply:
Using 'info', 'error', etc. make the log messages too long by adding redundant information (the process name is already added automatically by systemd).

If you use 'info' you end up with:
`Oct 05 15:31:58 localhost timeinit-buildtime.sh[834]: [timeinit-buildtime.sh][INFO] Setting system time from build time.`
instead of:
`Oct 05 15:09:41 localhost timeinit-buildtime.sh[1046]: [INFO] Setting system time from build time.`

Reply:
Adding the service name comes from the helper script being first used in the initramfs scripts - we may have to add some smartness to remove the service name if called with systemd running.

Reply:
I think it is still better to have a long message than having an explicit `[$LOGLEVEL] ` prefix to each message, that is just going to make it more confusing. If we do not like the format that `os-helpers-logging` produces we should fix it there & have it for everybody.

Reply:
@markcorbinuk I think we can live with long messages for a bit - let's create a ticket to address the helper library so it formats correctly both when used from the initrafms and a systemd service, but handle it separately from this PR.

Reply:
Sure, that's what I meant